tsconfig.json

```diff
@@ -2,12 +2,12 @@
   "compilerOptions": {
       "outDir": "./dist/",        // path to output directory
       "sourceMap": true,          // allow sourcemap support
-      "strictNullChecks": false,   // enable strict null checks as a best practice
```

Comment:
Why is this line deleted?

Reply:
Using strict should enable strictNullChecks by default.